Android/GoogleBillingClient/build.cake

```diff
@@ -43,11 +49,19 @@ Task("nuget")
 	});
 });
 
+Task ("ci")
+	.IsDependentOn("nuget")
+	;
+
 Task ("clean")
 	.Does (() =>
 {
 	if (DirectoryExists ("./externals/"))
-		DeleteDirectory ("./externals", true);
+		DeleteDirectory ("./externals", new DeleteDirectorySettings {
+											Recursive = true,
+											Force = true
+											}
+						);
```

Comment:
There looks like there is some weird indentation here, maybe mixed tabs & spaces?